### ISSUE TITLE
[1.18.2 Fabric v6.2] Display name doesn't support certain languages

### ISSUE THREAD

**Issue by Rad586** [TheComputerizer/Music-Triggers#124]
**Describe the bug**
When BOSS is included in mob trigger, only display name works. However, it crashes when I use the translated name of ender dragon in Chinese. 
**To Reproduce**
1. Paste text below in _main.toml_.
```
	[triggers.mob]
		level = "1"
		resource_name = [ "BOSS" "末影龙" ]
```
2. Enter the world and game crashes. 
**Additional context**
I believe that this bug would affect many languages besides Chinese(for example, Japanese), though it's not tested yet.

**Comment by TheComputerizer** [comment]
Can you attach the crash report

**Comment by Rad586** [comment]
Sorry, I made it wrong, it was just kicking me out from the single player world, not crashing the game.
[latest.log.txt](https://github.com/TheComputerizer/Music-Triggers/files/11967005/latest.log.txt)

**Comment by TheComputerizer** [comment]
Looks like its a network issue with a packet assuming a specific charset type. It should be relatively simple to fix and will be in 6.3

**Comment by TheComputerizer** [comment]
Fixed in 6.3, but also the BOSS case supports mob IDs too now which should be a good alternative in case things are still broken
